src/cobra/core/reaction.py

```diff
@@ -60,22 +69,32 @@ class Reaction(Object):
 
     Parameters
     ----------
-    id : string
-        The identifier to associate with this reaction
-    name : string
-        A human readable name for the reaction
-    subsystem : string
-        Subsystem where the reaction is meant to occur
-    lower_bound : float
-        The lower flux bound
-    upper_bound : float
-        The upper flux bound
+    id : str, default None
+        The identifier to associate with this reaction (default None).
+    name : str, optional, default ""
+        A human readable name for the reaction (default "").
+    subsystem : str, optional, default ""
+        Subsystem where the reaction is meant to occur (default "").
+    lower_bound : float, default 0
+        The lower flux bound (default 0).
+    upper_bound : float, optional, default None
+        The upper flux bound (default None).
```

Comment:
This is a new style and not in line with numpydoc https://numpydoc.readthedocs.io/en/latest/format.html#parameters, default values should be in the description.

```suggestion
    id : str or None, optional
        The identifier to associate with this reaction (default None).
    name : str, optional
        A human readable name for the reaction (default "").
    subsystem : str, optional
        Subsystem where the reaction is meant to occur (default "").
    lower_bound : float, optional
        The lower flux bound (default 0).
    upper_bound : float or None, optional
        The upper flux bound (default None).
```